app/js/components/common/Header.js

```diff
@@ -11,9 +11,13 @@ export class Header extends Component {
       locationTags: [],
       currentLocationTag: "",
       defaultLocation: "",
-      currentUser: ""
+      currentLogOutUrl: "empty",
+      currentwindowUrl: window.location.href.includes("modules-refapp") ?
+        '/openmrs/appui/header/logout.action?successUrl=openmrs':
+        '/openmrs-standalone/appui/header/logout.action?successUrl=openmrs-standalone',
     };
```

Comment:
We cannot hardcode /openmrs/... or /openmrs-standalone.../ into here.  @djazayeri or @rkorytkowski - can you comment on the proper way to get the context path?